src/drivers/actuators/vertiq_io/module.yaml

```diff
@@ -0,0 +1,271 @@
+module_name: Vertiq IO
+serial_config:
+    - command: vertiq_io telemetry ${SERIAL_DEV}
```

Comment:
Telemetry might be the wrong name for this. It comes from the DSHOt one where it's only coming back iirc. Maybe bus, uart, control, etc?